src/content/docs/network-performance-monitoring/get-started/network-monitoring-guidelines.mdx

```diff
@@ -0,0 +1,327 @@
+---
+title: Guidelines for network monitoring
+tags:
+  - Network monitoring
+  - Guidelines
+translate:
+  - kr
+  - jp
+metaDescription: Review guidelines for network monitoring.
+---
+
+import networkCommonArchitecture from 'images/network_diagram_common-architecture.webp'
+
+This document serves as an overview of the best practices available for deployment of the [KTranslate container](https://github.com/kentik/ktranslate) used in New Relic’s network monitoring capability.
+The guidance here is based on the recommendations of various subject matter experts combined with common patterns of deployment that customers are using in their production environments today.
+
+## Deployment considerations [#deployment-considerations]
+
+For the purposes of this document, we will be referencing an architecture that covers the following requirements:
+
+* SNMP polling and SNMP Traps collection
+* Syslog collection from network devices
+* Network Flow collection in NetFlow v5, NetFlow v9, IPFIX, and sFlow protocols
+* Support for multiple sites separated by large distance, geographically
+
+## Common architecture [#common-architecture]
+
+<SideBySide>
+
+  <Side>
+    This diagram reflects common networking architecture with the following:
+
+* **DC_01 (AMER)**: 
+    * Three containers on one host serving the `DC_01` location in New York City.
```

Comment:
Is New York City an appropriate city? I replaced "America" with a sample city to match the other bullet points.